SU2_CFD/include/driver_structure.hpp

```diff
@@ -355,12 +356,12 @@ class CSingleZoneDriver : public CDriver {
 
```

Comment:
Is CGeneralDriver working also for multizone, now? If so, I would change the comments here.